frontend/packages/employee-frontend/src/assets/i18n/fi.ts

```diff
@@ -812,7 +812,7 @@ export const fi = {
       newPartner: 'Uusi puoliso',
       editPartner: 'Puolison muokkaus',
       removePartner: 'Puolison poisto',
-      confirmText: 'Haluatko varmasti poistaa puolison?',
+      confirmText: 'Haluatko varmasti poistaa puolison? Puolison vaihtuessa merkitse edelliselle suhteelle loppumisaika ja lisää sen jälkeen uusi',
```

Comment:
Supernit: 
```suggestion
      confirmText: 'Haluatko varmasti poistaa puolison? Puolison vaihtuessa merkitse edelliselle suhteelle loppumisaika ja lisää sen jälkeen uusi puoliso',
```

would feel slightly more complete and clearer